src/core/entities/crux-spec.ts

```diff
@@ -0,0 +1,18 @@
+import { config } from "../../index";
+import globalAssetList from "../global-asset-list.json";
+export const CruxSpec = {
+    blockstack: class blockstack {
+        public static gaiaWriteURL: string = config.BLOCKSTACK.GAIA_HUB;
```

Comment:
Define ICruxBlockstackInfra in `core` which has fields `gaiaWriteURL` & `bnsNodes`.

Implement it in infrastructure `CruxDefaultBlockstackInfra` and refer to `config.BLOCKSTACK` there